UI/Services/DistributedSessionService.cs

```diff
@@ -121,7 +125,8 @@ public async Task<bool> SessionDataExistsAsync(string preKey = DefaultPreKey)
     public async Task<bool> AnySessionDataExistsAsync()
     {
         await LoadDataFromDistributedDataStore();
-        return _contextAccessor!.HttpContext!.Session!.Keys.Any();
+        var sessionKey = GetSessionKey(FormPostPreKey);
+        return _contextAccessor!.HttpContext!.Session!.Keys.Where(x => x != sessionKey).Any();
```

Comment:
We can rewrite this with the below:

`return _contextAccessor!.HttpContext!.Session!.Keys.Any(x => x != sessionKey);`

Reply:
Done